components/micro-gateway-cli/src/main/resources/templates/tokenServices.mustache

```diff
@@ -138,7 +138,51 @@ service tokenService on tokenListenerEndpoint, apiSecureListener {
         }
         var result = caller->respond(forwardedResponse);
         if (result is error) {
-           log:printError("Error when responding during the token endpoint request", err = result);
+            log:printError("Error when responding during the token endpoint request", err = result);
```

Comment:
print the full error

Reply:
tokenServices.mustache file contains these log:printError lines in several places. It is better to refactor this file to use new printError() method instead of log:printError.  Shall we fix this with another PR?

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/973